run-tests.php

```diff
@@ -450,13 +450,8 @@ function save_or_mail_results()
 $pass_option_n = false;
 $pass_options = '';
 
-$compression = 0;
 $output_file = $CUR_DIR . '/php_test_results_' . date('Ymd_Hi') . '.txt';
 
-if ($compression && in_array("compress.zlib", stream_get_filters())) {
-	$output_file = 'compress.zlib://' . $output_file . '.gz';
-}
-
```

Comment:
Well, obviously the question was really dumb. :)

I'll have a closer look at this PR within the next days. At least *some* clean up of the script seems indeed overdue.